File: app/src/main/java/com/another1dd/cybersportnews_materialviewpager/ParseTitle.java
package com.another1dd.cybersportnews_materialviewpager;

import android.os.AsyncTask;

import org.jsoup.Jsoup;
import org.jsoup.nodes.Document;
import org.jsoup.nodes.Element;
import org.jsoup.select.Elements;

import java.io.IOException;
import java.util.LinkedHashMap;


class ParseTitle extends AsyncTask<String, Void, LinkedHashMap<String[], String>> {

    @Override
    protected LinkedHashMap<String[], String> doInBackground(String... params) {

        LinkedHashMap<String[], String> hashMap = new LinkedHashMap<>();

        try {
            Document document = Jsoup.connect(params[0]).get();
            Elements elements = document.select(".b-news-list-item__media-body");

            for (Element element : elements) {
                Element title = element.select(".b-news-list-item__title").first();
                Element time = element.select(".time.b-news-list-item__time.grey-text").first();
                String[] temp = {title.text(), time.text()};
                Element element1 = element.select("a[href]").first();

                hashMap.put(temp, element1.attr("abs:href"));
            }
        } catch (IOException e) {
            hashMap.put(new String[]{"There is no internet connection", "Нет подключения к интернету"}, "http://www.cybersport.ru/news/");
            e.printStackTrace();
        }

        return hashMap;
    }
}

File: app/src/main/java/com/another1dd/cybersportnews/model/ParseTitle.kt
package com.another1dd.cybersportnews.model

import android.os.AsyncTask
import org.jsoup.Jsoup
import java.io.IOException
import java.util.*


class ParseTitle : AsyncTask<String, Void, LinkedHashMap<Array<String>, String>>() {

    override fun doInBackground(vararg params: String): LinkedHashMap<Array<String>, String> {

        val hashMap = LinkedHashMap<Array<String>, String>()

        try {
            val document = Jsoup.connect(params[0]).get()
            val elements = document.select(".b-news-list-item__media-body")

            for (element in elements) {
                val title = element.select(".b-news-list-item__title").first()
                val time = element.select(".time.b-news-list-item__time.grey-text").first()
                val temp = arrayOf(title.text(), time.text())
                val element1 = element.select("a[href]").first()

                hashMap.put(temp, element1.attr("abs:href"))
            }
        } catch (e: IOException) {
            hashMap.put(arrayOf("There is no internet connection", "Нет подключения к интернету"), "http://www.cybersport.ru/news/")
            e.printStackTrace()
        }

        return hashMap
    }
}